react/Checkbox/Checkbox.js

```diff
@@ -24,11 +23,13 @@ export default class Checkbox extends Component {
   static propTypes = {
     id: PropTypes.string.isRequired,
     label: PropTypes.node.isRequired,
+    value: PropTypes.string,
```

Comment:
Not sure we should expose `value` from Checkbox. I don't think it's technically valid.

Reply:
I didn't originally add it, but realised as I was preparing the PR that it's actually valid: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Same for radio buttons too: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

Reply:
As an example of where this is useful, you might have a large checklist and want to share a single event handler across every checkbox instance, checking `event.target.value` in the callback.